reference/api/pagination.md

```diff
@@ -0,0 +1,94 @@
+# Pagination
+
+_Child route of the [settings route](/reference/api/settings.md)._
+
+This route allows you to configure the pagination settings for an index.
+
+Pagination settings can also be updated directly through the [global settings route](/reference/api/settings.md#update-settings) along with the other settings.
+
+To learn more about paginating search results with Meilisearch, refer to our [dedicated guide](/learn/advanced/pagination.md).
+
+::: warning
+Updating the settings means overwriting the default settings of Meilisearch. You can reset to default values using the `DELETE` routes.
+:::
+
+## Get pagination settings
+
+<RouteHighlighter method="GET" route="/indexes/{index_uid}/settings/pagination"/>
+
+Get the pagination settings of an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+
+### Example
+
+<CodeSamples id="get_pagination_settings_1" />
+
+#### Response: `200 OK`
+
+```json
+{
+  "maxTotalHits": 1000
+}
+```
+
+### Returned fields
+
+#### `maxTotalHits`
+
+The maximum number of results Meilisearch can return.
+
+## Update pagination settings
+
+<RouteHighlighter method="PATCH" route="/indexes/{index_uid}/settings/pagination"/>
+
+Partially update the pagination settings for an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
+
+### Body
+
+#### `maxTotalHits`
+
+**Type:** integer
+**Default value:** `1000`
```

Comment:
Do we need to mention that 1000 is the recommended value? [Spec (lines 31-33)](https://github.com/meilisearch/specifications/pull/157/files#diff-ec4edaf515880ccb83c711f72245fb90498bea850b4256a25d0d215c8550fe31):
>The value of 1000 ensures good performance and prevents malicious users from scraping documents from a Meilisearch instance.
>Increasing this value can degrade performance as well as expose the data of an instance to scrapping.

Reply:
Good question. I think I would rather us not say that it is recommended (there are many legitimate reasons for changing this value), but mentioning it may have an effect on performance sounds reasonable.